src/main/java/racinggame/model/car/Car.java

```diff
@@ -0,0 +1,65 @@
+package racinggame.model.car;
+
+import java.util.Objects;
+import racinggame.model.move.MoveStrategy;
+import racinggame.model.position.Position;
+
+public class Car {
+    private static final String RACE_STATE_PER_CAR_DELIMITER = " : ";
+
+    private Position position;
+    private CarName name;
+
+    public Car(String name) {
+        this.position = new Position();
+        this.name = new CarName(name);
+    }
```

Comment:
```suggestion
    public Car(String name) {
        this(new Position(), new CarName(name));
    }
    
    public Car(Position position, CarName name) {
        this.position = position;
        this.name = name;
    }
```
필드 값을 초기화하는 주 생성자를 추가하고, 부 생성자가 주 생성자를 호출하는 방식으로 구현하는 방식으로 접근해보면 어떨까?
이와 관련해서는 2차 강의 내용 중 참고